rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -282,9 +289,36 @@ def get_poap_airdrop_data(name: str, data_dir: Path) -> dict[str, Any]:
     return data_dict
 
 
+def is_airdrop_claimed(
+        address: ChecksumEvmAddress,
+        amount: FVal,
+        asset: Asset,
+        database: DBHandler,
```

Comment:
since this logic is executed inside a nested loop consider creating `events_db = DBHistoryEvents(database)` outside in `def check_airdrops` and use it here. for the cursor you can do `events_db.db.conn.read_ctx()` since is an attribute. Doing this you avoid creating an object `NUMBER_OF_AIDROPS * NUMBER OF ADDRESSES` times